projects/miragesdk/examples/https-unikernel/README.md

```diff
@@ -0,0 +1,70 @@
+## https-unikernel example service
```

Comment:
Does this already work as solo5 "unikernel"?
If not, it might be better to mention that?